src/store/socket.js

```diff
@@ -0,0 +1,16 @@
+import io from 'socket.io-client';
+import { POSTS_SUCCESS } from '../actions/PostActions';
+
+const token = '';
+
+const socket = io('http://localhost:5000/notes');
```

Comment:
Should be in file with constants
`'http://localhost:5000/'` , `notes`
And use template strings